pkg/postgres/schema/process_indicators.go

```diff
@@ -1,4 +1,7 @@
-// Code generated by pg-bindings generator. DO NOT EDIT.
+// Code generated by pg-bindings generator.
```

Comment:
nit:  `// Code originally generated by pg-bindings generator.`

Reply:
Making this change causes style-check to run on this file, and that reports style errors for automatically generated code.

Reply:
Done